apollo-compiler/src/test/kotlin/com/apollostack/compiler/GraphQLCompilerTest.kt

```diff
@@ -23,7 +24,7 @@ class GraphQLCompilerTest {
     val actual = actualFileFor("hero_name", "TestQuery")
     val expected = expectedFileFor("hero_name", "TestQuery")
 
-    compiler.write(irFileFor("hero_name", "TestQuery"))
+    compiler.write(irFileFor("hero_name", "TestQuery"), outputDir)
```

Comment:
How about to extract this as util function:

```
private fun compile(pkg: String, queryFileName: String) {
  compiler.write(irFileFor("hero_name", "TestQuery"), outputDir)
}
```

So in tests it will be much shorter:

```
compile("hero_name", "TestQuery")
```